pkg/api/unversioned/well_known_labels.go

```diff
@@ -28,3 +28,20 @@ const (
 	LabelOS   = "beta.kubernetes.io/os"
 	LabelArch = "beta.kubernetes.io/arch"
 )
+
+// Role labels are applied to Nodes to mark their purpose.
+const (
+	// LabelRole is a label applied to a Node as a hint that it has a particular purpose (defined by the value).
+	LabelRole = "kubernetes.io/role"
+
+	// LabelKubeadmAlphaRole is a label that kubeadm applies to a Node as a hint that it has a particular purpose.
+	LabelKubeadmAlphaRole = "kubeadm.alpha.kubernetes.io/role"
```

Comment:
Why do we need 2 different labels?

There isn't enough explanation here for anyone to understand what they should do with these. Please expand the comment above the block.